cdap-ui/app/directives/complex-schema/array-schema/array-schema.html

```diff
@@ -24,12 +24,14 @@
     <div class="clearfix">
       <div class="input-column select-only no-type"
            ng-class="{ 'tab-header': ArraySchema.items.nested }">
-        <div class="select-wrapper">
-          <select class="form-control"
-                  ng-model="ArraySchema.items.displayType"
-                  ng-options="option as option for option in ArraySchema.SCHEMA_TYPES"
-                  ng-change="ArraySchema.changeType()">
-          </select>
+        <div ng-class="{ 'tab-header-inner': ArraySchema.items.nested }">
```

Comment:
we don't need an ng-class for this. just set the class to ```tab-header-inner``` and in the css, nest tab-header-inner inside tab-header